### PR TITLE
Users' account recovery

### PR DESCRIPTION
**Pull request by Oyinbrakemigrace**
[resetPassword.webm](https://github.com/annonymousauthority/P2PCoder/assets/117949160/6ef0339f-ca07-428f-b713-4486f555e8d8)

 Users are now able to recover account from form.

### CODE REVIEW

File: development/src/components/userLogin_comp.js [hunk 1]
```diff
@@ -65,102 +74,126 @@ function UserLoginComp() {
...
+                  inputId="password"
+                  placeholder="Enter password"
+                  handleInputChange={passwordChange}
+                  inputValue={password}
+                />
+                {passwordError && (
+                  <p className="text-sm text-red-500">{passwordError}</p>
+                )}
+                <button
+                  className="float-right font-bold text-[#5F5BD7]"
```
Comment (annonymousauthority) [inline development/src/components/userLogin_comp.js:166]: remove the `font-bold` from the forgot password

File: development/src/components/userLogin_comp.js [hunk 2]
```diff
@@ -65,102 +74,126 @@ function UserLoginComp() {
...
+                  onClick={forgotPassword}
```
Comment (annonymousauthority) [inline development/src/components/userLogin_comp.js:167]: Please utilize the conditionals for the route. It should be in the like we have in `welcome_comp.js` lines 12-33.
It shouldn't be a true/false conditional route.

Comment (annonymousauthority) [reply development/src/components/userLogin_comp.js:167]: The route can be `/?view=recoveraccount`

**Review by annonymousauthority — CHANGES_REQUESTED** [review]
All modals should be on their own routes of value `/?view=${name}`. Forgot password isn't a child of login component

**Comment by Oyinbrakemigrace** [comment]
I noticed the eye icon in the input element was not aligned to the center so i fixed it.
[Uploading ddabd66f-4f8b-4b7e-82c3-6c8ae3b0a773.webm…]()

**Review by 7malikk — DISMISSED** [review]

**Review by Remi-dee — DISMISSED** [review]

**Review by annonymousauthority — APPROVED** [review]
LG

**Comment by Oyinbrakemigrace** [comment]
[passwordRecovery.webm](https://github.com/annonymousauthority/P2PCoder/assets/117949160/4bf56ce1-a2f2-4b73-ba9e-7417d77ced9a)

File: development/src/components/userLogin_comp.js (hunk 2 above)
Comment (Oyinbrakemigrace) [reply development/src/components/userLogin_comp.js]: Done

File: development/src/components/userLogin_comp.js (hunk 1 above)
Comment (Oyinbrakemigrace) [reply development/src/components/userLogin_comp.js]: Done

**Review by Remi-dee — APPROVED** [review]

**Pull request merged**